ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-bgp-evpn-mpls.yml

```diff
@@ -37,6 +37,9 @@ router_bgp:
     - ip_address: 192.168.255.2
       peer_group: EVPN-OVERLAY-PEERS
   address_family_evpn:
+    next_hop_mpls_resolution_ribs:
+      primary_rib: "tunnel-rib colored system-colored-tunnel-rib"
+      fallback_rib: "system-connected"
```

Comment:
We try to avoid doing more of these now. Are the models very very complex? Do you have some example?